cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -1445,14 +1546,17 @@ class device_row_comparator {
 
     template <typename Element,
               CUDF_ENABLE_IF(not cudf::is_equality_comparable<Element, Element>() and
-                             (not has_nested_columns or not cudf::is_nested<Element>())),
+                             (not has_nested_columns or not cudf::is_nested<Element>()) and
```

Comment:
```suggestion
                             (not has_nested_columns) and
```
It seems to be a redundant check

Reply:
I think just keeping `cudf::is_nested<Element>()` should work? But I see this pattern again at https://github.com/tgujar/cudf/blob/hash-occupancy/cpp/include/cudf/table/experimental/row_operators.cuh#L498. Unsure if missing anything here

Reply:
> I think just keeping cudf::is_nested<Element>() should work?

Correct.